prototype_source/prototype_index.rst

```diff
@@ -207,7 +207,7 @@ Prototype features are not available as part of binary distributions like PyPI o
    prototype/fx_graph_mode_ptq_dynamic.html
    prototype/fx_graph_mode_ptq_static.html
    prototype/graph_mode_dynamic_bert_tutorial.html
-   prototype/quantization_in_pytorch_2_0_export_tutorial.html
+   prototype/quantization_in_pytorch_2_export_tutorial.html
```

Comment:
@jerryzh168 which file is this referring to? I couldn't find it

Reply:
this is the pt2e_quantizer file

Reply:
could you add the other two files to this list as well?